packages/manager/src/features/Databases/DatabaseDetail/DatabaseNetworking/DatabaseConnectionPools.tsx

```diff
@@ -0,0 +1,215 @@
+import { useDatabaseConnectionPoolsQuery } from '@linode/queries';
+import {
+  Button,
+  CircleProgress,
+  ErrorState,
+  Hidden,
+  styled,
+  Typography,
+} from '@linode/ui';
+import { useTheme } from '@mui/material/styles';
+import { Pagination } from 'akamai-cds-react-components/Pagination';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeaderCell,
+  TableRow,
+} from 'akamai-cds-react-components/Table';
+import React from 'react';
+
+import { ActionMenu } from 'src/components/ActionMenu/ActionMenu';
+import { usePaginationV2 } from 'src/hooks/usePaginationV2';
+
+import {
+  getActionMenuWrapperStyles,
+  makeSettingsItemStyles,
+} from '../../shared.styles';
+
+import type { Database } from '@linode/api-v4';
+import type { Action } from 'src/components/ActionMenu/ActionMenu';
+
+interface Props {
+  database: Database;
+  disabled?: boolean;
+}
+
+const ConnectionPoolActionMenuStyledWrapper = styled(TableCell, {
+  label: 'ConnectionPoolActionMenuStyledWrapper',
+})(({ theme }) => getActionMenuWrapperStyles(theme));
+
+export const DatabaseConnectionPools = ({ database }: Props) => {
+  const { classes } = makeSettingsItemStyles();
+  const theme = useTheme();
+  const poolLabelCellStyles = {
+    flex: '.5 1 20.5%',
+  };
+
+  const pagination = usePaginationV2({
+    currentRoute: '/databases/$engine/$databaseId/networking',
+    initialPage: 1,
+    preferenceKey: `database-connection-pools-pagination`,
+  });
+
+  const {
+    data: connectionPools,
+    error: connectionPoolsError,
+    isLoading: connectionPoolsLoading,
+  } = useDatabaseConnectionPoolsQuery(database.id, true, {
+    page: pagination.page,
+    page_size: pagination.pageSize,
+  });
+
+  const PAGE_SIZES = [25, 50, 75, 100];
+  const MIN_PAGE_SIZE = 25;
+
+  const results = connectionPools?.results;
+
+  const connectionPoolActions: Action[] = [
+    {
+      onClick: () => null,
+      title: 'Edit', // TODO: UIE-9395 Implement edit functionality
+    },
+    {
+      onClick: () => null, // TODO: UIE-9430 Implement delete functionality
+      title: 'Delete',
+    },
+  ];
+
+  if (connectionPoolsLoading) {
+    return <CircleProgress />;
+  }
+
+  if (connectionPoolsError) {
+    return (
+      <ErrorState errorText="There was a problem retrieving your connection pools. Refresh the page or try again later." />
+    );
+  }
+
+  return (
+    <>
+      <div className={classes.topSection}>
+        <div className={classes.sectionTitleAndText}>
+          <div className={classes.sectionTitle}>
+            <Typography variant="h3">
+              Manage PgBouncer Connection Pools
+            </Typography>
+          </div>
+          <Typography sx={{ maxWidth: '500px' }}>
+            Manage PgBouncer connection pools to minimize the use of your server
+            resources.
+          </Typography>
+        </div>
```

Comment:
We can just simplify this by using a Stack and adding `display: 'block'` to the topSection theme.breakpoints.down('md') style 
```suggestion
        <Stack>
          <Typography variant="h3">
            Manage PgBouncer Connection Pools
          </Typography>
          <Typography sx={{ maxWidth: '500px' }}>
            Manage PgBouncer connection pools to minimize the use of your server
            resources.
          </Typography>
        </Stack>
```